### DISCUSSION TITLE
Asking Help on reviewing my config.

### DISCUSSION THREAD

**Discussion by lidgnulinux** [vim/vim#14761] [Q&A] [answered]
About a year ago, I'm using these lines of config to automate on markdown writing. Unfortunatelly when I want to use it again, it's no longer works. For example, when I'm in insert mode and press ALT+i, nothing happen (the "__" won't show up).  Here my config :    
```    
autocmd FileType markdown inoremap <M-b> ****
	\ | inoremap <M-i> __
 	\ | inoremap <M-u> []()
 	\ | vnoremap <M-b> c**<C-r>"**<Esc>
 	\ | vnoremap <M-i> c_<C-r>"_<Esc>
 	\ | vnoremap <M-c> c`<C-r>"`<Esc>    
```
What could be wrong there ? I'm using Ubuntu and slackware. Thanks.

**Comment by zeertzjq** [comment]
What keyboard layout and terminal emulator are you using?

**Reply by lidgnulinux** [reply]
I'm using US keyboard layout, for the terminal, I'm using foot and sometimes gvim terminal.

The config still works on neovim though. I'm wondering why it doesn't work on Vanilla vim.

**Reply by lidgnulinux** [reply]
Oh , apparently I need to add `set filetype=` manually. It works now. Thanks and solved.

**Reply by lidgnulinux** [reply, marked as answer]
Oh , apparently I need to add `set filetype=` manually. It works now. Thanks and solved.
